styles.css

```diff
@@ -0,0 +1,91 @@
+.bell-icon {
+    position: absolute;
+    right: 0;
+    top: 10px;
+  }
+  
+  .micophone-icon {
+    position: absolute;
+    right: 50px;
+    top: 10px;
+  }
+  
+  .Account {
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
+header {
+    width: 100%;
+    height: 70px;
+    background-color: #fff;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+  }
+  .logo img{
+    width: 80px;
+    height: 80px;
+    object-fit: contain;
+    display: inline-block;
+    background-color: black;
+    
+  }
+ .search-form{
+    display: flex;
+    align-items: center;
+ }
+ .search-form .search-container{
+display: flex;
+align-items: center;
+position: relative;
+justify-content: center;
+ }
+ 
+ .search-form .search-container input[type="text"] {
+    width: calc(100% - 50px);
+    height: 40px;
+    padding: 0 10px;
+    border: 2px solid #ddd;
+    border-radius: 4px;
+    font-size: 14px;
+  }
+  #search-query {
+    width: 60%;
+    height: 40px;
+    padding: 0 10px;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    font-size: 14px;
+  }
+  #clear-button {
+    width: 40px;
+    height: 40px;
+    background-color: transparent;
+    border: 0;
+    border-radius: 4px;
+    cursor: pointer;
+    color: #666;
+    font-size: 20px;
+  }
+  .bell.img {
+    position: absolute;
+    left: 100px;
+    top: 10px;
+  }
+  
+  .micophone {
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
+  
+  .Account {
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
```

Comment:
can you please finish styling your code in order to provide a more appealing website